src/components/App/App.css

```diff
@@ -0,0 +1,51 @@
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+#root{
+	font-size: 12px;
+}
+
+.appWrap {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #f5f5f5;
+}
+.headerWrap {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  width: 85%;
+  /* background-color: #f5f5f5; */
+  position: sticky;
+  top: 0;
+  z-index: 3;
+  /* border-bottom: 1px solid #221e22; */
+}
+.logo {
+  width: 30rem;
+}
```

Comment:
Thanks for getting some styling done! the cards and overall look of the app is very nice!

Reply:
Initial styling is looking great! I like the responsive touches to when a user hovers over different elements